README.md

```diff
@@ -6,8 +6,29 @@ at any point, and most data is dumped every 12 hours, after analysis and stat st
 
 ## Installation
 
-Coming Soon
+1. Clone this repo on your server into the folder **jstats-server**
+2. Create a database called **jstats-server**
+3. Create a table with this code
+```
+CREATE TABLE IF NOT EXISTS `jos_jstats` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `unique_id` varchar(32) NOT NULL,
+  `php_version` varchar(255) NOT NULL,
+  `db_type` varchar(255) NOT NULL,
+  `db_version` varchar(255) NOT NULL,
+  `cms_version` varchar(255) NOT NULL,
+  `server_os` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB  DEFAULT CHARSET=utf8 ;
+```
+
+4. Run ```composer install``` from the jstats-server folder to install the dependencies
+5. Run ```composer update``` from the jstats-server folder to update the lock file
```

Comment:
People definitely should NOT do this :P The whole point is that the lock file in github is the version which works. and running ```composer update``` is the responsibility of us as long  as we are sure that the server works with the updated requirements